### PR TITLE
Update linea

### PR DESCRIPTION
**Pull request by croll83**
## Description

Added rpcUrls.infura
Added contracts.multicall3

## Additional Information

The public endpoint is rate limited to 20req/sec based on source IP. If dApps rely on it, they can get 429s. In such a case better to switch over Infura provider

ENS/address: @croll83.eth

### CODE REVIEW

**Review by jxom — APPROVED** [review]

**Comment by croll83** [comment]
Hey team, can someone help here with this PR? 

How to fix the failing check?

**Comment by aaronmgdr** [comment]
> Hey team, can someone help here with this PR?
> 
> How to fix the failing check?

I got the same one on my PR.

File: packages/chains/src/lineaTestnet.ts
Comment (tmm) [inline packages/chains/src/lineaTestnet.ts:1]: Please run `pnpm lint:fix` over this file. Few formatting differences that should be fixed either by running that script or with git hooks (automatic if installed).

Comment (croll83) [reply packages/chains/src/lineaTestnet.ts:1]: tried but it seems that there are no changes after the lint:fix:

image

should I close this PR and create a new branch to fix this?

**Pull request merged**